packages/flutter/test/cupertino/route_test.dart

```diff
@@ -1513,7 +1513,7 @@ void main() {
     await tester.pumpAndSettle();
 
     expect(semantics, includesNodeWith(
-      actions: <SemanticsAction>[SemanticsAction.tap, SemanticsAction.dismiss],
+      actions: <SemanticsAction>[SemanticsAction.tap],
```

Comment:
it looks like the dismiss is removed from the modalbarrier, is there a reason why?